manifests/s/Swift/Toolchain/5.7.3/Swift.Toolchain.installer.yaml

```
@@ -21,10 +21,7 @@ Dependencies:
   - PackageIdentifier: Git.Git
 Installers:
 - Architecture: x64
-  InstallerUrl: https://download.swift.org/swift-5.7.2-release/windows10/swift-5.7.2-RELEASE/swift-5.7.2-RELEASE-windows10.exe
-  InstallerSha256: F523F8D364135738D9E7419F987A654A15159424434A13FC480079BCFCE78C75
-  AppsAndFeaturesEntries:
-    - DisplayVersion: 5.7.0
-      DisplayName: Swift Developer Package for Windows x86_64
+  InstallerUrl: https://download.swift.org/swift-5.7.3-release/windows10/swift-5.7.3-RELEASE/swift-5.7.3-RELEASE-windows10.exe
+  InstallerSha256: 179b44cf7f1ce5c74dc864207119464ff41526a2cc2257cfd2ccef65bc25b231
```

```suggestion
  InstallerSha256: 179b44cf7f1ce5c74dc864207119464ff41526a2cc2257cfd2ccef65bc25b231
  ProductCode: '{71166b25-5db3-454d-b4d8-cf96b0caf289}'
  AppsAndFeaturesEntries:
    - DisplayName: Swift Developer Package for Windows x86_64
```
We can keep DisplayName as it helps WinGet for better upgrade matching. Suggested ProductCode for the same reason.